src/Components/pages/Config.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import {fire, auth, firestore} from '../../firebase.js';
+import {setUpRoom,getIndustries,getSectors,getMarketCaps,initSymbols} from '../firebase-access.jsx'
+import AsyncSelect from 'react-select/async';
+import Select from 'react-select'
+
+const ROUNDS = [
+    { value: 1, label: '1 Round' },
+    { value: 2, label: '2 Rounds' },
+    { value: 3, label: '3 Rounds' },
+    { value: 4, label: '4 Rounds' },
+    { value: 5, label: '5 Rounds' },
+    { value: 6, label: '6 Rounds' },
+    { value: 7, label: '7 Rounds' },
+  ]
+class Config extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+          Industry:null,
+          MarketCap:null,
+          Sector:null,
+          Rounds:null
+        };
+        this.submitHandler = this.submitHandler.bind(this);
+        this.getIndList = this.getIndList.bind(this);
+        this.getSectorList = this.getSectorList.bind(this);
+        this.getMarketCapList = this.getMarketCapList.bind(this);
+      }
+      // async componentDidMount() {  
+      //     this.setState({
+      //         Industries: await getIndustries(firestore),
+      //         Sectors: await getSectors(firestore),
+      //         MarketCaps: await getMarketCaps(firestore),
+      //     });
+      // }
+  
+      getIndList = async()=>{
+         
+          let IndList =await getIndustries(firestore);
+          return IndList;
+      }

```

Comment:
just keep industry in, it's not that long and otherwise Ind could stand for a bunch of other things

```suggestion
      getIndustryList = async () => {
          let IndustryList = await getIndustries(firestore);
          return IndustryList;
      }

```